### ISSUE TITLE
Typings out of whack

### ISSUE THREAD

**Issue by garthk** [novemberborn/ksuid#11]
This works (JavaScript):

```js
const INSTANCE_ID = require('ksuid').randomSync().string;
```

This transpiles as correct, given your typings, but doesn't work (TypeScript):

```ts
import * as ksuid from 'ksuid';

const INSTANCE_ID = ksuid.KSUID.randomSync().string;
```
→ JavaScript:
```js
const ksuid = require("ksuid");
const INSTANCE_ID = ksuid.KSUID.randomSync().string;
```

Workaround (TypeScript):

```ts
const INSTANCE_ID: string = require('ksuid').randomSync().string;
```

Thanks for the module, though! Totally saved me having to do it myself.

**Comment by novemberborn** [comment]
Yea, that should be `import KSUID = require('ksuid')` but that doesn't work either.

Could you check with https://github.com/novemberborn/ksuid/pull/12?

**Comment by garthk** [comment]
```
import { parse } from 'ksuid';
```
→
```
Module '"/path/to/node_modules/ksuid/index"' resolves to a non-module entity and cannot be imported using this construct
```
… and without a `default` export, this can't work either:
```
import ksuid from 'ksuid';
```
… but this does work:
```
import ksuid = require('ksuid');
/* … and, later … */
ksuid.parse(request_id) // got typing! yay!
```

**Comment by novemberborn** [comment]
Yup, since this module is written as CJS you need the `import foo = require('foo')` syntax.

To clarify, this solves the issue?

I'm on holiday at the moment but will merge and publish if this solves the problem for you, when I get the time.

**Comment by garthk** [comment]
Yes, solved. Thanks.

**Comment by novemberborn** [comment]
Thanks @garthk. Released as 1.1.2.

**Comment by zer09** [comment]
@novemberborn I tried to use like this ```import ksuid = require('ksuid');``` but I am still getting this error
```
Import assignment cannot be used when targeting ECMAScript modules. Consider using 'import * as ns from "mod"', 'import {a} from "mod"', 'import d from "mod"', or another module format instead.
```

But if I use it like this ```import * as ksuid from 'ksuid';``` I am getting this
```
Module '"/.../node_modules/ksuid/index"' resolves to a non-module entity and cannot be imported using this construct.
```

Any suggestion what to do? thanks.
